### ISSUE TITLE
More docs?

### ISSUE THREAD

**Issue by rulatir** [schmittjoh/php-manipulator#10]
Where can I find a more complete documentation?

**Comment by harikt** [comment]
Have you seen http://jmsyst.com/libs/php-manipulator

**Comment by rulatir** [comment]
Yes, which is why I asked about _more_ docs, like a complete API reference.

**Comment by rulatir** [comment]
How to create a matcher array for skipUntil()? What does even go in there - class names? Some constants? For example, I want to capture all strings for automated __("wrapping") and converting interpolations to sprintf, but there are several string node types - one for single quoted, one for double quoted, and probably still another for double quoted but without any interpolations.

Where is the reference for all this?

**Comment by harikt** [comment]
API is the source code you download. You can either create your own ( apigen or phpdocumentator ) or read the source code. I don't know whether it has a hosted api though.

Hope that helps :) .
